requirements.in

```diff
@@ -0,0 +1,6 @@
+https://github.com/Abstract-Tech/derex.runner/tarball/master#egg=derex.runner
+jinja2
+pip-tools
```

Comment:
I wouldn't list this among the requirements. It's needed only to update `requirements.txt`.

If we remove it from here the only negative effect AFAIU is a slight inconvenience for the developer doing the version upgrade.

Also I think this is semantically wrong: pip-tools is not a requirement for this project. It's a tool that we use to maintain the list of requirements.